snapcraft/tests/test_commands_clean.py

```diff
@@ -25,11 +25,32 @@
 from snapcraft.main import main
 from snapcraft.internal import (
     pluginhandler,
+    project_loader,
     states,
 )
 from snapcraft import tests
 
 
+class _CompareExpectedStates():
```

Comment:
I think I would rather have this be a method ala `assertStatesEqual(state1, state2)` b extending from the `TestCase` class (call it `StateTestClass` if out of ideas).